config/environments/staging.rb

```diff
@@ -46,15 +46,17 @@
     config.cache_store = :mem_cache_store, ENV['LAGO_MEMCACHE_SERVERS'].split(',')
 
   elsif ENV['LAGO_REDIS_CACHE_URL'].present?
-    config.cache_store = :redis_cache_store, {
-      url: ENV['LAGO_REDIS_CACHE_URL'],
-      error_handler: ->(method:, returning:, exception:) {
-        Rails.logger.error(exception.message)
-        Rails.logger.error(exception.backtrace.join("\n"))
+    config.cache_store =
+      :redis_cache_store,
+      {
+        url: ENV['LAGO_REDIS_CACHE_URL'],
+        error_handler: lambda { |method:, returning:, exception:|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `returning`. If it's necessary, use `_` or `_returning` as an argument name to indicate that it won't be used. (https://rubystyle.guide#underscore-unused-vars)